test/models/v2/servers.spec.ts

```diff
@@ -46,4 +68,28 @@ describe('Servers model', function () {
       expect(servers.has('production')).toEqual(false);
     })
   })
+
+  describe('.filterBySend()', function () {
+    it('should return all servers with channels with subscribe operation', function () {
+      const servers = new Servers(serverItems);
+      expect(servers.filterBySend()).toEqual([serverItems[2], serverItems[3]]);
```

Comment:
This test is failing due to some kind of sourcery comparing the message JSON value with the actual message inside the AsyncAPI parsed document. See https://github.com/asyncapi/parser-js/blob/next-major/src/models/v2/message.ts#L67

It works if comparison is done with `JSON.stringify()`, which obviously is not gonna happen since that method preserves order of keys ending in a not a fear comparison.